community/OM-Operator-Management/cnv-samples/fleet-user-rolebinding-policy/acm-kubevirt-edit-fleet.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: acm-kubevirt-edit-fleet
+  namespace: default
+  annotations:
+    policy.open-cluster-management.io/categories: AC Access Control
+    policy.open-cluster-management.io/controls: AC-3 Access Enforcement
+    policy.open-cluster-management.io/description: This is the virt.k8s.io:edit ClusterRole for all CNV managed clusters
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+spec:
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-rolebinding
+        spec:
+          namespaceSelector:
+            exclude:
+              - kube-*
+            include:
+              - default
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: rbac.authorization.k8s.io/v1
+                kind: ClusterRoleBinding
+                metadata:
+                  name: acm-kubevirt-edit
+                roleRef:
+                  name: kubevirt.io:edit
+                  apiGroup: rbac.authorization.k8s.io
+                  kind: ClusterRole
+                subjects:
+                  - name: jnpacker
+                    apiGroup: rbac.authorization.k8s.io
+                    kind: User
+                  - name: davidvossel
+                    apiGroup: rbac.authorization.k8s.io
+                    kind: User
```

Comment:
Should these be `virtuser01` and `virtuser02`?

Reply:
I'm going to remove this whole section at this point and just keep it to the policy.